app/helpers/text_helper.rb

```diff
@@ -0,0 +1,7 @@
+module TextHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
```

Comment:
Line is too long. [93/80]<br>Replace class var @@renderer with a class instance var.